app/ide-desktop/lib/dashboard/src/authentication/src/index.tsx

```diff
@@ -4,7 +4,7 @@
 /** This import is unused in this file, but React doesn't work without it, under Electron. This is
  * probably because it gets tree-shaken out of the bundle, so we need to explicitly import it.
  * Unlike all other imports in this project, this one is not `camelCase`. We use `React` instead of
- * `react` here. This is because if the import is named any differently then React doesn't get
+ * `React` here. This is because if the import is named any differently then React doesn't get
  * included in the final bundle. */
```

Comment:
Seems like the second half of this comment no longer valid since we're not using the lower-case import anymore. It also seems like the below comment and lints might need to be updated?

Reply:
yeah, i think the entire comment can be removed - because the lints will warn when `React` is not in scope anyway